src/lib/validators.ts

```diff
@@ -33,10 +36,7 @@ export async function validateFilePath(
 	try {
 		const stat = await statAsync(path);
 		if (!stat.isFile()) {
-			if (raise) {
-				throw new Error('Error: ' + path + ' is not a file');
-			}
-			return false;
+			throw new Error('Error: ' + path + ' is not a file');
```

Comment:
TypedError.

Reply:
Validators are dropped, as we are now using schema everywhere.